control/control.rnc

```diff
@@ -1057,6 +1057,29 @@ update = element update {
 
 # update
 
+configuration_management = element configuration_management {
+    element type { "salt" | "puppet" }? &
+    element master { text }? &
+    element auth_attempts { INTEGER }? &
+    element auth_time_out { INTEGER }? &
+    element keys_url { text }? &
+    element enable_services { BOOLEAN }? &
+
+    # Salt specific
+    element formulas_roots { text }? &
+    element pillar_root { text }? &
+    element pillar_url { text }? &
+    element states_roots { text }? &
```

Comment:
In the example you have formulas_roots and states_roots as lists.

Reply:
Oh, my fault. Fixed.